.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,63 @@
+name: Build and test libslac
+on:
+  pull_request: {}
+  workflow_dispatch:
+    inputs:
+      runner:
+        description: Which runner to use
+        type: choice
+        default: 'ubuntu-22.04'
+        required: true
+        options:
+          - 'ubuntu-22.04'
+          - 'large-ubuntu-22.04-xxl'
+
+jobs:
+  lint:
+    name: Lint
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
```

Comment:
should use workflow_dispatch input and ubuntu-22.04 as default see https://github.com/EVerest/everest-core/blob/main/.github/workflows/build_and_test.yaml